vanilla/model_views.py

```diff
@@ -83,16 +83,11 @@ def get_form_class(self):
         if self.form_class is not None:
             return self.form_class
 
-        if self.model is not None:
-            if self.fields is None:
-                msg = "'Using GenericModelView (base class of '%s) without setting" \
-                    "either 'form_class' or the 'fields' attribute is pending deprecation."
-                warnings.warn(msg % self.__class__.__name__,
-                              PendingDeprecationWarning)
```

Comment:
Could you determine which Django version this changes in?
Then we can figure out the right way to document this change and determine if it's okay for us to make the change at this point.